src/db/types.ts

```diff
@@ -1,7 +1,32 @@
 import { ObjectId } from "mongodb";
+export interface User {
+	_id?: ObjectId;
+	isBlind: boolean;
```

Comment:
this strikes me as a lil weird phrasing, how about `isBlindMode` or something? Someone who isn't totally blind might not want to check "I'm blind" on the frontend, but they might want "Blindness mode" or something less personally-judgemental. I know this is just the backend side but it reads a little weird to me.

Reply:
I totally agree. I had no idea what to call this so I'm glad your suggestion sounds better! Thanks!